ui/runServer.sh

```diff
@@ -10,9 +10,10 @@ SCALA_VERSION=${SCALA_VERSION:-2.12}
 PROJECT_BASE_DIR="../../.."
 #management jars are currently needed to access ProcessManagers
 FLINK_ENGINE_JAR=$PROJECT_BASE_DIR/engine/flink/management/target/scala-${SCALA_VERSION}/nussknacker-flink-manager.jar
+FLINK_PERIODIC_ENGINE_JAR=$PROJECT_BASE_DIR/engine/flink/management/target/scala-${SCALA_VERSION}/nussknacker-flink-periodic-manager.jar
 STANDALONE_ENGINE_JAR=$PROJECT_BASE_DIR/engine/standalone/engine/target/scala-${SCALA_VERSION}/nussknacker-standalone-manager.jar
 
-export CLASSPATH="../target/scala-${SCALA_VERSION}/nussknacker-ui-assembly.jar:$FLINK_ENGINE_JAR:$STANDALONE_ENGINE_JAR"
+export CLASSPATH="../target/scala-${SCALA_VERSION}/nussknacker-ui-assembly.jar:$FLINK_ENGINE_JAR:$FLINK_PERIODIC_ENGINE_JAR:$STANDALONE_ENGINE_JAR"
```

Comment:
I think it's not needed ATM, as periodic manager currently can be used only via custom extension (we don't expose any ProcessManagerProvider via ServiceLoader)